### PR TITLE
Run tests locally on windows

### PR DESCRIPTION
**Pull request by studioj**
as requested PR for having tests runnable on windows and having the 2.7 build pass

### CODE REVIEW

**Comment by studioj** [comment]
should i remove the 2.7 part? I see you disabled the build (but maybe thats because it was failing

**Comment by rbanffy** [comment]
The 2.7 removal was mostly because of the official Python abandonment of 2.7, so it makes sense to no longer worry too much about it. I think we can remove 2.7 now.

File: requirements_dev.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-bandit
+bandit==1.6.2; python_version < "3"
```
Comment (rbanffy) [inline requirements_dev.txt:1]: Let's not bother supporting 2.x

File: tests/test_pip_chill.py [hunk 2]
```diff
@@ -10,12 +10,17 @@
...
 import sys
 import unittest
 
-from pip_chill import pip_chill
+from pip_chill import pip_chill, cli
 from pip_chill.pip_chill import Distribution
 
+PIP_CHILL_CLI_FULL_PATH = cli.__file__
+PYTHON_EXE_PATH = sys.executable
+is_windows = any(platform.win32_ver())
```
Comment (rbanffy) [inline tests/test_pip_chill.py:22]: I'm reading TravisCI's Windows documentation and trying to figure out how it works. After this is merged, we should think about enabling Windows CI too.

File: requirements_dev.txt (hunk 1 above)
Comment (studioj) [reply requirements_dev.txt]: fixed

**Comment by studioj** [comment]
shoudl be ready to merge

**Comment by rbanffy** [comment]
From a quick look, it seems the Travis tests are failing because of the new crypto modules written in Rust. I'll need to add a Rust compiler to the config.

**Comment by studioj** [comment]
@rbanffy  one remaining :-)

**Comment by studioj** [comment]
@rbanffy  i dont know how important that dist: bionic was => but removing it made it work

**Comment by studioj** [comment]
I think this is still a valid PR @rbanffy  I reopened.

**Comment by rbanffy** [comment]
Agreed. If there is a need for this, we need to reassess.

**Comment by studioj** [comment]
Explain reassess please. :) Can I do something? 
Or do you asume the project complete?

**Pull request closed without merging**